docs/About.md

```diff
@@ -29,12 +29,31 @@ will need to clone the `dev` branch:
 git clone -b dev https://github.com/iriusrisk/startleft.git
 ```
 
-And then deploy the documentation using the provided `docker-compose` file inside the `deployment` folder:
+And then launch the StartLeft documentation
+
+
+## Launch StartLeft documentation by Dockerfile
+
+Deploy the documentation using the provided `docker-compose` file inside the `deployment` folder:
```

Comment:
Should the docs be updated to use the newer "docker compose" command instead of "docker-compose"? It is mentioned three times.

Reply:
You are right that we can found users with docker installed from debian/ubintu packages docker.io and users with docker installed from the docker.com packages (e.g: docker-ce)
I'm going to modify the docs to cover both.
Thanks!